pyqtgraph/parametertree/xml_base.py

```diff
@@ -0,0 +1,84 @@
+from abc import abstractmethod
+from xml.etree import ElementTree as ET
+
+from ..parametertree import Parameter
+
+
+class XMLParameter:
+    """ Mixin class defining methods to serialize Parameter object to XML string """
+
+    @staticmethod
+    def common_options_from_xml(el: ET.Element) -> dict:
+        """ Extract XML element into a dictionary
+
+        Parameters
+        ----------
+        el: ET.Element
+            The xml element to extract information from
+
+        Returns
+        -------
+        dict
+        """
+        basic_options = {
+            "name": el.get('name'),
+            "type": el.get('type'),
+            "title": el.get('title', el.tag),
+            "visible": el.get('visible', '1') == '1',
+            "removable": el.get('removable', '0') == '1',
+            "readonly": el.get('readonly', '0') == '1',
+            "tip": el.get('tip', ""),
+            "show_pb": el.get('show_pb', '0') == '1'
```

Comment:
what is "show_pb" used for?

Reply:
sorry it has been left here by mistake (it is related to a specific Parameter I will PR later)